src/en/news/blog/2023/v18-2-0-reef-released/index.md

```diff
@@ -0,0 +1,953 @@
+>    ```
+
+1. Set the `noout` flag for the duration of the upgrade. (Optional, but recommended.)
+
+      ```
+      ceph osd set noout
+      ```
+
+2. Upgrade monitors by installing the new packages and restarting the monitor daemons. For example, on each monitor host
+
+      ```
+      systemctl restart ceph-mon.target
+      ```
+
+   Once all monitors are up, verify that the monitor upgrade is complete by looking for the `reef` string in the mon map. The command
+
+      ```
+      ceph mon dump | grep min_mon_release
+      ```
+
+   should report:
+
+      ```
+      min_mon_release 18 (reef)
+      ```
+
+   If it does not, that implies that one or more monitors hasn't been upgraded and restarted and/or the quorum does not include all monitors.
+
+3. Upgrade `ceph-mgr` daemons by installing the new packages and restarting all manager daemons. For example, on each manager host,
+
+      ```
+      systemctl restart ceph-mgr.target
+      ```
+
+   Verify the `ceph-mgr` daemons are running by checking `ceph -s`:
+
+      ```
+      ceph -s
+
+      ...
+      services:
+      mon: 3 daemons, quorum foo,bar,baz
+      mgr: foo(active), standbys: bar, baz
+      ...
+      ```
+
+4. Upgrade all OSDs by installing the new packages and restarting the ceph-osd daemons on all OSD hosts
+
+      ```
+      systemctl restart ceph-osd.target
+      ```
+      
+5. Upgrade all CephFS MDS daemons. For each CephFS file system,
+
+   1. Disable standby_replay:
+
+         ```
+         ceph fs set <fs_name> allow_standby_replay false
+         ```
+
+   2. **Only** if upgrading from a Pacific <=16.2.5:
+
+         ```
+         ceph config set mon mon_mds_skip_sanity true
+         ```
+
+   3. Reduce the number of ranks to 1. (Make note of the original number of MDS daemons first if you plan to restore it later.)
+
+         ```
+         ceph status # ceph fs set <fs_name> max_mds 1
+         ```
+
+   4. Wait for the cluster to deactivate any non-zero ranks by periodically checking the status
+
+         ```
+         ceph status
+         ```
+
+   5. Take all standby MDS daemons offline on the appropriate hosts with
+
+         ```
+         systemctl stop ceph-mds@<daemon_name>
+         ```
+
+   6. Confirm that only one MDS is online and is rank 0 for your FS
+
+         ```
+         ceph status
+         ```
+
+   7. Upgrade the last remaining MDS daemon by installing the new packages and restarting the daemon
+
+         ```
+         systemctl restart ceph-mds.target
+         ```
+
+   8. Restart all standby MDS daemons that were taken offline
+
+         ```
+         systemctl start ceph-mds.target
+         ```
+
+   9. Restore the original value of `max_mds` for the volume
+
+         ```
+         ceph fs set <fs_name> max_mds <original_max_mds>
```

Comment:
@ljflores you need to unset skip_sanity down here.

Reply:
Added